examples/3-mutation/src/Dog.re

```diff
@@ -56,22 +58,28 @@ let make =
       [|id|],
     );
 
-  let (_, executeMutation) = Hooks.useMutation(~query=Mutations.treatDog);
+  let (_, executeLikeMutation) =
+    Hooks.useMutation(~request=Mutations.LikeDog.make(~key=id, ()));
+
+  let (_, executeTreatMutation) =
+    Hooks.useMutation(
+      ~request={
```

Comment:
Is the idea here to show a way to use `useMutation` without `graphql_ppx`? If so, I like it!